src/main/protobuf/soc/gameState.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto2";
+
+package protos.soc;
+
+import "soc/board.proto";
+import "soc/inventory.proto";
+import "soc/playerState.proto";
+
+message GameState {
+    required Board board = 1;
+    repeated PlayerState player_states = 2;
+    required int32 num_development_cards_left = 3;
+    required Player current_player = 4;
+    required bool can_roll_dice = 5;
+    required bool can_play_card = 6;
+    required ResourceSet resource_bank = 7;
+    optional DevelopmentCardSet development_card_deck = 8;
+    optional PossibleHands possible_hands = 9;
```

Comment:
Can we remove the possible_hands stuff from this PR? My understanding from our conversation is that this would go in its own RPC, that would not return a `GameState` object, but rather a specific, perspective view object we would define.